fedrec/utilities/io_utils.py

```diff
@@ -67,6 +89,9 @@ def dash_separated_ints(value):
 
 
 def dash_separated_floats(value):
+    """ 
+    Returns dash seperated list of float values.
```

Comment:
You can re-write it as
```suggestion
    This method takes a list of float values as input and returns the dash-separated list of values.   
```
You can add an example for it as well. and check the pep8 as well in case it exceeds the value.